araproc/analysis/hilbert.py

```diff
@@ -0,0 +1,97 @@
+# this calculates Hilbert envelope SNR for each channel and their average
+
+import numpy as np
+from scipy.signal import hilbert
+
+from araproc.analysis import snr
+from araproc.framework import waveform_utilities as wfu
+
+
+
+def get_envelope(waveform):
```

Comment:
Sorry can this function be moved to `AraProc/araproc/framework/waveform_utilities.py`?